README.md

```diff
@@ -3477,6 +3478,32 @@ You can view the future development items list in the [github project issues](ht
 ## Contributing
 See [contributing guide](.github/CONTRIBUTING.md)
 
+<a name="development"></a>
+## Development
```

Comment:
lets move this whole thing under 'editor support' and maybe create sub section for vim and vscode (vim already exists).